chiptopher/guet
# Pair timeout

## Overview
Make it so that you must reset the current pair every 24 hours.


## Testing Instructions
```
guet set cb "chris boyer" email@localhost
~~~ wait 24 hours ~~~
git commit -m "Commit message"
```
You should then see the output:
```
Must set pairs again
```
